Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/ActiveClientDiagnosticTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos.Tests
+{
+    using System;
+    using System.Threading.Tasks;
+    using System.Xml.Serialization;
+    using VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class ActiveClientDiagnosticTest
+    {
+        private const string ConnectionString = "AccountEndpoint=https://localtestcosmos.documents.azure.com:443/;AccountKey=425Mcv8CXQqzRNCgFNjIhT424GK99CKJvASowTnq15Vt8LeahXTcN5wt3342vQ==;";
+
+        private int prevActiveClients = 0;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            this.prevActiveClients = CosmosClient.NumberOfActiveClients;
```

Comment:
Why is this needed? You could just simply set it to 0, right?

```suggestion
            CosmosClient.NumberOfActiveClients = 0;
```